hack/jenkins/e2e.sh

```diff
@@ -1,230 +0,0 @@
-  export KUBE_GCE_ZONE="us-central1-f"
-  export FAIL_ON_GCP_RESOURCE_LEAK="true"
-elif [[ ${JOB_NAME} =~ ^kubernetes-.*-gke ]]; then
-  export KUBERNETES_PROVIDER="gke"
-  export ZONE="us-central1-f"
-  # By default, GKE tests run against the GKE test endpoint using CI Cloud SDK.
-  # Release jobs (e.g. prod, staging, and test) override these two variables.
-  export CLOUDSDK_BUCKET="gs://cloud-sdk-build/testing/staging"
-  export CLOUDSDK_API_ENDPOINT_OVERRIDES_CONTAINER="https://test-container.sandbox.googleapis.com/"
-  export FAIL_ON_GCP_RESOURCE_LEAK="true"
-elif [[ ${JOB_NAME} =~ ^kubernetes-.*-aws ]]; then
-  export KUBERNETES_PROVIDER="aws"
-  export E2E_MIN_STARTUP_PODS="1"
-  export KUBE_AWS_ZONE="us-west-2a"
-  export MASTER_SIZE="m3.medium"
-  export NODE_SIZE="m3.medium"
-  export NUM_NODES="3"
-fi
-
-# Set environment variables based on upgrade jobs
-if [[ ${JOB_NAME} =~ ^kubernetes-upgrade ]]; then
-  # Upgrade jobs bounce back and forth between versions; just force
-  # it to always get the tars of the version it wants to test.
-  export JENKINS_FORCE_GET_TARS="y"
-  export FAIL_ON_GCP_RESOURCE_LEAK="false"
-  if [[ "${KUBERNETES_PROVIDER}" == "gce" ]]; then
-    export NUM_NODES=5
-  fi
-  if [[ "${JOB_NAME}" =~ step1 ]]; then
-    export E2E_TEST="false"
-    export E2E_DOWN="false"
-  elif [[ "${JOB_NAME}" =~ step2 ]]; then
-    export E2E_OPT="--check_version_skew=false"
-    export E2E_UP="false"
-    export E2E_DOWN="false"
-    export GINKGO_TEST_ARGS="--ginkgo.focus=Kubectl"
-  elif [[ "${JOB_NAME}" =~ step3 ]]; then
-    export E2E_OPT="--check_version_skew=false"
-    export E2E_UP="false"
-    export E2E_DOWN="false"
-    export GINKGO_TEST_ARGS="--ginkgo.focus=\[Feature:Upgrade\].*upgrade-master"
-  elif [[ "${JOB_NAME}" =~ step4 ]]; then
-    export E2E_OPT="--check_version_skew=false"
-    export E2E_UP="false"
-    export E2E_DOWN="false"
-  elif [[ "${JOB_NAME}" =~ step5 ]]; then
-    export E2E_OPT="--check_version_skew=false"
-    export E2E_UP="false"
-    export E2E_DOWN="false"
-    export GINKGO_TEST_ARGS="--ginkgo.focus=\[Feature:Upgrade\].*upgrade-cluster"
-  elif [[ "${JOB_NAME}" =~ step6 ]]; then
-    export E2E_OPT="--check_version_skew=false"
-    export E2E_UP="false"
-    export E2E_DOWN="false"
-  elif [[ "${JOB_NAME}" =~ step7 ]]; then
-    # TODO(15011): these really shouldn't be (very) version skewed, but
-    # because we have to get ci/latest again, it could get slightly out of
-    # whack.
-    export E2E_OPT="--check_version_skew=false"
-    export E2E_UP="false"
-  fi
-fi
-
-# Define environment variables based on the Jenkins project name.
-# NOTE: Not all jobs are defined here. The hack/jenkins/e2e.sh in master and
-# release branches defines relevant jobs for that particular version of
-# Kubernetes.
-case ${JOB_NAME} in
-
-  # PR builder
-
-  # Runs a subset of tests on GCE in parallel. Run against all pending PRs.
-  kubernetes-pull-build-test-e2e-gce)
```

Comment:
I already made the change on PR builder. I need to get that Jenkins instance into source control soon.